azkaban-common/src/main/java/azkaban/executor/ExecutorManagerAdapter.java

```diff
@@ -88,6 +88,18 @@ public ExecutableFlow getExecutableFlow(int execId)
 
   public List<ExecutableFlow> getRunningFlows() throws IOException;
 
+  /**
+   * <pre>
+   * Returns All running with executors and queued flows
+   * Note, returns empty list if there isn't any running or queued flows
+   * </pre>
+   *
+   * @return
+   * @throws IOException
+   */
+  public List<Pair<ExecutableFlow, Executor>> getActiveFlowsWithExecutor()
```

Comment:
I really think we should create a new Class, or recycle some rarely used Class (say the executionReference) to represent this data structure so that we can replace the first and second call with some meaningful getter names.